hadoop-ozone/ozone-manager/src/test/java/org/apache/hadoop/ozone/om/request/snapshot/TestOMSnapshotCreateRequest.java

```diff
@@ -256,7 +256,7 @@ public void testEmptySnapshotRenamedKeyTable() throws Exception {
     renameKey("key1", "key2");
```

Comment:
@aswinshakil  can we add an additional test case for renaming the directory and verify.

Reply:
Done. Added it.